test/functional/apps/console/_console.ts

```diff
@@ -122,5 +122,34 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
         });
       });
     });
+
+    describe('multiple requests output', () => {
+      const sendRequests = async () => {
+        await PageObjects.console.enterRequest('\n PUT test-index');
+        await PageObjects.console.enterRequest('\n DELETE test-index');
+        await PageObjects.console.selectAllRequests();
+        await PageObjects.console.clickPlay();
+      };
+      beforeEach(async () => {
+        await PageObjects.console.clearTextArea();
+      });
+      it('should contain comments starting with # symbol', async () => {
+        await sendRequests();
+        await retry.try(async () => {
+          const response = await PageObjects.console.getResponse();
+          log.debug(response);
+          expect(response).to.contain('# PUT test-index 200 OK');
+          expect(response).to.contain('# DELETE test-index 200 OK');
+        });
+      });
+      it('should display status badges', async () => {
+        // This request should fail
+        await PageObjects.console.enterRequest('\n GET test-index');
+
+        await sendRequests();
+        await retry.waitFor('success badge', () => PageObjects.console.hasSuccessBadge());
```

Comment:
to be honest, I don’t really like this approach, in case of an error we will try to find this element again and again. I think it would be more correct to determine that we have updated the output of the console and then check the result `retry.waitFor`